packages/block/src/types.ts

```diff
@@ -87,6 +89,46 @@ export interface BlockData {
   uncleHeaders?: Array<HeaderData>
 }
 
+/**
+ * An object with the block's data represented as strings.
+ */
+export interface JsonBlock {
+  /**
+   * Header data for the block
+   */
+  header?: string
+  transactions?: string
+  uncleHeaders?: string
+}
+
+/**
+ * An object with the block header's data represented as strings.
+ */
+export interface JsonHeader {
+  parentHash?: string
+  uncleHash?: string
+  coinbase?: string
+  stateRoot?: string
+  transactionsTrie?: string
+  receiptTrie?: string
+  bloom?: string
+  difficulty?: string
+  number?: string
+  gasLimit?: string
+  gasUsed?: string
+  timestamp?: string
+  extraData?: string
+  mixHash?: string
+  nonce?: string
+}
```

Comment:
Update: ah, ok, got it, this is used subsequently in the `JsonBlock` interface.

Reply:
Yeah I was just following the pattern set up in the tx refactor, I'm not entirely sure the specific reason to have them optional instead of required (maybe more valuable in the tx context, or some general rpc thing), but should be fine.